### PR TITLE
fix: add account mfe patch to show account page

### PR DESCRIPTION
**Pull request by hinakhadim**
Issue:
With Tutor v18 and Tutor Indigo v18.1.0, users are unable to view the accounts page. Tutor Indigo v18.1.0 has installation of custom indigo header and footer in learning, dashboard, profile, discussion and account MFE. All MFEs are compatible with indigo header and footer except accounts MFE. The reason is:

In redwood:
Most of the MFEs using ----> footer v14

footer is installed via @openedx/frontend-slot-footer package
footer is imported like import FooterSlot from '@openedx/frontend-slot-footer';
Account MFE -----> footer v13

footer is installed via @edx/frontend-component-footer
footer is imported like import { FooterSlot } from '@edx/frontend-component-footer';
Our indigo footer is created from footer v14 and we are installing it in all MFEs (account MFE has footer v13). More details can be found on https://github.com/openedx/frontend-app-account/issues/1071

This PR adds the patch in account-mfe which fixes the above issue

Solves https://github.com/overhangio/tutor/issues/1111

### CODE REVIEW

File: tutormfe/patches/mfe-dockerfile-pre-npm-build-account [hunk 1]
```diff
@@ -0,0 +1,4 @@
+RUN git config --global user.email "tutor@overhang.io" \
+  && git config --global user.name "Tutor"
```
Comment (DawoudSheraz) [inline tutormfe/patches/mfe-dockerfile-pre-npm-build-account:2]: nit: do we need to configure git user here?

File: changelog.d/20240820_112614_hina.khadim_account_mfe_patch.md [hunk 2]
```diff
@@ -0,0 +1 @@
+- [BugFix] Add upstream footer-slot patch in account-mfe (by @hinakhadim)
```
Comment (DawoudSheraz) [inline changelog.d/20240820_112614_hina.khadim_account_mfe_patch.md:1]: the changelog is not correct.

File: tutormfe/patches/mfe-dockerfile-pre-npm-build-account (hunk 1 above)
Comment (hinakhadim) [reply tutormfe/patches/mfe-dockerfile-pre-npm-build-account:2]: Yes, we need to do so. We are getting below error if we aren't configuring git user:
```bash
1.542 Committer identity unknown
1.542 
1.542 *** Please tell me who you are.
1.542 
1.542 Run
1.542 
1.542   git config --global user.email "you@example.com"
1.542   git config --global user.name "Your Name"
1.542 
1.542 to set your account's default identity.
1.542 Omit --global to set the identity only in this repository.
1.542 
1.542 fatal: unable to auto-detect email address (got 'root@buildkitsandbox.(none)')
------
Dockerfile:113
--------------------
 111 |     ARG ENABLE_NEW_RELIC=false
 112 |     
 113 | >>> RUN curl -fsSL https://github.com/openedx/frontend-app-account/commit/16fb3a1bb4c00d2b60ec3686001814461437e3a5.patch | git am
 114 |     
 115 |     ######## account (dev)
```

**Comment by hinakhadim** [comment]
Closing this PR due to merge conflicts caused by the included patch, which is preventing the patch from being applied successfully and resulting in errors.

```bash
Applying: fix: import `FooterSlot` from `frontend-slot-footer` package (#1048)
error: patch failed: package-lock.json:10
error: package-lock.json: patch does not apply
error: patch failed: package.json:28
error: package.json: patch does not apply
error: patch failed: src/index.jsx:11
error: src/index.jsx: patch does not apply
Patch failed at 0001 fix: import `FooterSlot` from `frontend-slot-footer` package (#1048)
```

**Pull request closed without merging**